public/video-ui/src/components/VideoUpload/VideoTrail.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import {YouTubeEmbed, youTubeUrl} from '../utils/YouTubeEmbed';
+import Icon from '../Icon';
+import {getProcessingStatus} from '../../services/YoutubeApi';
+import _ from 'lodash';
+
+const VIDEO_WIDTH = 320;
```

Comment:
could we set this to `100%` and control the dimensions of the containing element in css?